tests/settings/common.py

```diff
@@ -13,6 +13,7 @@
     'django.contrib.staticfiles',
     'django.contrib.humanize',
     'premis_event_service',
+    'tests',
```

Comment:
I'm not sure why this is here. Doesn't seem to be the norm for us, at least not that I've noticed. Perhaps you did something different in this?